doc/api/permissions.md

```diff
@@ -1,4 +1,28 @@
-# Policies
+# Permissions
+
+This section exposes security features available to be adopted in a
+Node.js application. The available scopes are:
+
+* [Resource-based permissions](#resource-based-permissions)
+* [Process-based permissions](#process-based-permissions)
+
+Resource-based permissions stands for the managment of modules using
+policies. A policy can guarantee which module/resource is available
+during the application execution.
+
+Process-based permissions stands for the management of resources such
+as _File System_ or _Network_. A permission can be configured to restrict
+access to specific resources, for instance, one can restrict access to
+all the _File System_ write.
+
+Both permissions can be used together to provide a safer environment.
+
+**Note**: if you find a potential security vulnerability on Node.js,
+refer to our [Security Policy][].
```

Comment:
I would describe process before resource/module throughout this page. Also I took a pass at conforming to our style guide; others such as @Trott are more expert on that. Basically, minimize formatting such as italics or bold and things like “Note:”, if I remember correctly.

```suggestion
* [Process-based permissions](#process-based-permissions) control the Node.js process's access to
resources such as the file system or the network. The resource can be
entirely allowed or denied, or actions related to it can be controlled; 
for example, you can allow file system reads while denying writes.

* [Module-based permissions](#module-based-permissions) control which files or URLs are available
to other modules during application execution. This can be used to
control what modules can be accessed by third-party dependencies,
for example.

Both types of permissions can be used together to provide a
safer environment. If you find a potential security vulnerability,
please refer to our [Security Policy][].
```